prow/repoowners/repoowners.go

```diff
@@ -78,6 +78,9 @@ type cacheEntry struct {
 }
 
 type Client struct {
+	DirBlacklistByRepo  map[string]sets.String
```

Comment:
These fields do not need to be exported, so we could pass in the configuration to the `NewClient` call and have it set up private fields in the `Client` struct like it does for the logger and cache.